portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+  <dependency>
```

Comment:
```suggestion
      <dependency>
```
Please fix the indentation so that the `<dependency></dependency>` tags are aligned

Reply:
I missed that, thank you.